scriptmodules/emulators/amiberry.sh

```diff
@@ -106,12 +106,14 @@ function configure_amiberry() {
 
     # move config / save folders to $md_conf_root/amiga/amiberry
     local dir
-    for dir in conf savestates screenshots; do
+    for dir in conf nvram savestates screenshots; do
         moveConfigDir "$md_inst/$dir" "$md_conf_root/amiga/amiberry/$dir"
     done
 
-    # symlink cd32.nvr from $md_inst/data to "$md_conf_root/amiga/amiberry
-    moveConfigFile "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/cd32.nvr"
+    # check for cd32.nvr and move it to $md_conf_root/amiga/amiberry/nvram
+    if [[ -f "$md_conf_root/amiga/amiberry/cd32.nvr" ]]; then
+	mv "$md_conf_root/amiga/amiberry/cd32.nvr" "$md_conf_root/amiga/amiberry/nvram/"
```

Comment:
You have a tab indentation here - please use 4 spaces.

Reply:
Really done a botch job here... Just for a (very) simple `if` statement :( Apologies once again. Hope this time it will be ok.